at-tanvo/Controllers/session_controller.rb

```diff
@@ -0,0 +1,72 @@
+require 'bcrypt'
+require './Model/employee.rb'
+class SessionController
+  def login
+    system('clear')
+    puts 'xxxxxxx Login xxxxxxx'
+    puts 'Username: '
+    username = gets.chomp
+    puts 'Password:'
+    password = gets.chomp
+    result = Employee.new.check_user(username)
+    unless result.count == 0
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/aaa8a77d0d40b4c4a78140482c3fa32e36501a75/at-tanvo/Controllers/session_controller.rb#L12"></span>Style/NumericPredicate: Use <code>result.count.zero?</code> instead of <code>result.count == 0</code>. (https://github.com/rubocop-hq/ruby-style-guide#predicate-methods)</td>
    </tr>
  </tbody>
</table>